Frontend/src/Pages/Services.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import Service from "../comopnents/Service/Service";
+
+const Services = () => {
+  return (
+    <>
+      <Service />
+      <a href=""></a>
```

Comment:
Since you have routing it's better to use the Link tag from react-router-dom

Reply:
Ok sir